week-1-ts-ecosystem/w01e01-typescript-basics/src/types.ts

```diff
@@ -0,0 +1,13 @@
+interface Address {
+  street: string;
+  no: number;
+}
+
+interface User {
+  name: string;
+  age: number;
+  email: string;
+  address?: Address;
+}
+
+export {Address, User}
```

Comment:
To jest model, i to bym rozbił
zobacz, np. folder:
`model`
a w nim 2 osobne pliki żeby widzieć jaki jest model:

`model/Address.ts`
`model/User.ts`